tests/contrib/funders/test_funders_jsonschema.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2022 CERN.
+#
+# Invenio-Vocabularies is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see LICENSE file for more
+# details.
+
+"""Funders JSONSchema tests."""
+
+import pytest
+from jsonschema.exceptions import ValidationError
+
+from invenio_vocabularies.contrib.funders.api import Funder
+
+
+@pytest.fixture(scope="module")
+def schema():
+    """Returns the schema location."""
+    return "local://funders/funder-v1.0.0.json"
+
+
+def validates(data):
+    """Validates funder data."""
+    Funder(data).validate()
+
+    return True
+
+
+def fails(data):
+    """Validates funder data."""
+    pytest.raises(ValidationError, validates, data)
+    return True
+
+
+def test_valid_full(appctx, schema):
+    data = {
+        "$schema": schema,
+        "id": "funder-1",
+        "name": "Test funder",
+        "pid": {
+            "pk": 1,
+            "status": "R",
+            "pid_type": "funderid",
+            "obj_type": "funder"
+        },
+        "scheme": "Test scheme"
+    }
+
+    assert validates(data)
+
+
+def test_valid_empty(appctx, schema):
```

Comment:
moderate: shouldn't there be? for the "relations" there should not, but at service level we should not allow insertion of empty funder records.